src/resolvers/antdv.ts

```diff
@@ -175,6 +175,15 @@ export interface AntDesignVueResolverOptions {
    * @default false
    */
   importLess?: boolean
+
+  /**
+   * should resolve `ant-design-vue' icons 
+   * 
+   * required package `@ant-design/icons-vue`
+   * 
+   * @default undefined
```

Comment:
```suggestion
   * @default false
```

Reply:
In fact if I don't pass the `resolveIcons` , JS will stealth convert it to ` false ` . so I just need to change the comment here to `@default false`. right ?

Reply:
They are basically the same in this situation. Having `false` is more clear for user to read as the type is more like "optional boolean"